Magic™Editor.user.js

```diff
@@ -3494,21 +3495,20 @@
         // This is the styling for the diff output.
         $('body').append('<style>' +
                          '.difftitle {' +
-                         '    color: rgb(34, 34, 34);' +
+                         '    color: var(--black);' +
```

Comment:
Again, here `rgb(34, 34, 34)` is the same as `#222222`; the Stacks.css `--black` variable is `#ffffff` in dark mode and `#0c0d0e` in light.

Reply:
The actual title colour is set to `--black-700` - I will use that instead.